indico/htdocs/js/indico/modules/eventmanagement.js

```diff
@@ -37,6 +37,30 @@
             headerTemplate: '',
             sortList: [[1, 0]]
         });
+
+        $('.js-count-label').qbubble({
```

Comment:
this is a super generic name, but then inside the function you have `contributions` which is much less generic... `s/contributions/items/`?